### DISCUSSION TITLE
How I supposed to install .msixbundle from releases without having AppInstaller installed

### DISCUSSION THREAD

**Discussion by ITaluone** [microsoft/winget-cli#4113] [Q&A] [answered]
I feel really stupid to ask, how I should initially install the AppInstaller from the release page _without_ having any sort of software which installs `.msix` /`msixbundle` files?

How this is supposed to work?

Thanks for any advice

**Comment by Trenly** [comment, marked as answer]
In PowerShell you should be able to use `Add-AppxPackage <path-to-file>` to install it directly. You can also skip the download entirely and use `Add-AppxPackage https://aka.ms/getwinget` for the latest stable release or `Add-AppxPackage https://aka.ms/getwingetpreview` for the latest pre-release

**Reply by ITaluone** [reply]
Ahh thank you 🎉🎉

Works like a charm
